### ISSUE TITLE
Indented code blocks have extra whitespace

### ISSUE THREAD

**Issue by X9VoiD** [OpenTabletDriver/opentabletdriver.github.io#134]
![image](https://github.com/OpenTabletDriver/opentabletdriver.github.io/assets/10338031/a65a537a-b38a-4b7d-8ce7-fb2947b36a9d)

![image](https://github.com/OpenTabletDriver/opentabletdriver.github.io/assets/10338031/9519d14e-73c1-442a-9178-b7f18b5e36e8)

**Comment by gonX** [comment]
I don't think this is easy to fix. It looks like the extra whitespace exists to support diff-style markers, I think. I ran into the lack of consistency here when trying to implement #155, as it would occasionally not add the single whitespace, and at other times fill too much whitespace.
